che-plugin-openshift-ext-client/src/main/resources/org/eclipse/che/ide/ext/openshift/client/OpenshiftLocalizationConstant.properties

```diff
@@ -102,8 +102,10 @@ deploy.project.window.variables.add.tooltip=Add new variable
 deploy.project.window.labels.add.tooltip=Add new label
 deploy.project.success=Project <b>{0}</b> successfully deployed to OpenShift
 
-invalid.openshift.project.name.error=Name must be a valid DNS label
-invalid.che.project.name.error=Name must be a valid Che project name
+invalid.openshift.project.name.error=Name must be a valid project name
```

Comment:
We came with idea to show "Invalid name (?)", and when user moves mouse over "?", it will display tooltip:
"Project names must be in range 3 to 63 symbols, and may only contain lower-case letters, numbers, and dashes. They may not start or end with a dash."